com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/LightLoopDef.hlsl

```diff
@@ -18,6 +18,7 @@ struct HDShadowContext { float unused; };
 struct LightLoopContext
 {
     int sampleReflection;
+    float3 positionWS;
```

Comment:
add a comment about why we need it here.  could be under ifdef #ifdef UNITY_ATMOSPHERIC_SCATTERING_INCLUDED